resources/icons/vscode/data/double_extension_to_icon_name_map.rs

```diff
@@ -0,0 +1,2 @@
+[
```

Comment:
In resources directory, to make this easily editable. Technically these aren't `.rs` files, just mappings written in the form or `rs` files.